vnext/Shared/HermesShim.cpp

```diff
@@ -59,6 +70,14 @@ static void EnsureHermesLoaded() noexcept {
     s_dumpSampledTraceToFile = reinterpret_cast<decltype(s_dumpSampledTraceToFile)>(
         GetProcAddress(s_hermesModule, dumpSampledTraceToFileSymbol));
     VerifyElseCrash(s_dumpSampledTraceToFile);
+
+    s_makeHermesRuntimeWithWER = reinterpret_cast<decltype(s_makeHermesRuntimeWithWER)>(
+        GetProcAddress(s_hermesModule, makeHermesRuntimeWithWERSymbol));
+    VerifyElseCrash(s_makeHermesRuntimeWithWER);
+
+    s_hermesCrashHandler =
```

Comment:
I think this means apps that were created with let's say RNW 0.66, would now crash when upgraded to 67 because the export is not found in _their_ version of Hermes
Can we fail gracefully instead?

Reply:
This whole shim thing feels weird, not sure what it's for exactly. But I suspect apps would need to version match RNW and Hermes, right? We don't support N-1 / N+1 with random Hermes binaries.

Reply:
It's to be kosher with WACK

WACK tries to prevent you from shipping with missing DLLs, and will fail validation if you link to a function exported by a DLL not present. This is still safe when the DLL is delay-loaded, but WACK does not know about it.

We wanted a single Microsoft.ReactNative DLL, but the ability to opt-out of packaging Hermes DLL. This lead to loading the DLL at runtime, from its publicly exported function names (the mangled ones).